### ISSUE TITLE
Howdy working for sudo authentication but not for login

### ISSUE THREAD

**Issue by haji-ali** [boltgolt/howdy#337]
I am getting an `Unknown error: 1` when I try to login with howdy. However authenticating sudo works as expected. Looking into `/var/log/auth.log`, I see the following log

```
Apr 15 09:21:22 thinkpad /lib/security/howdy/pam.py[27348]: Traceback (most recent call last):
Apr 15 09:21:22 thinkpad /lib/security/howdy/pam.py[27348]:   File "/lib/security/howdy/pam.py", line 68, in pam_sm_authenticate
Apr 15 09:21:22 thinkpad /lib/security/howdy/pam.py[27348]:     return doAuth(pamh)
Apr 15 09:21:22 thinkpad /lib/security/howdy/pam.py[27348]:   File "/lib/security/howdy/pam.py", line 62, in doAuth
Apr 15 09:21:22 thinkpad /lib/security/howdy/pam.py[27348]:     pamh.conversation(pamh.Message(pamh.PAM_ERROR_MSG, "Unknown error: " + str(status)))
Apr 15 09:21:22 thinkpad /lib/security/howdy/pam.py[27348]: PamException: Conversation error
```

Any idea how to resolve this?
I've searched for similar issues already, and my issue has not been reported yet.

Linux distribution (if applicable): Ubuntu

Howdy version: 2.5.1

**Comment by ramiel** [comment]
I have the same error. Why is this closed? Did you solve it?

**Comment by ramiel** [comment]
Ok, I solved following #243
